terraform/README.md

```diff
@@ -0,0 +1,73 @@
+# SD-Core NMS K8s Terraform Module
+
+This SD-Core NMS K8s Terraform module aims to deploy the [sdcore-nms-k8s charm](https://charmhub.io/sdcore-nms-k8s) via Terraform.
+
+## Getting Started
+
+### Prerequisites
+
+The following software and tools needs to be installed and should be running in the local environment.
+
+- `microk8s`
+- `juju 3.x`
+- `terrafom`
+
+### Deploy the sdcore-nms-k8s charm using Terraform
+
+Make sure that `storage`, `multus` and `metallb` plugins are enabled for Microk8s:
+
+```console
+sudo microk8s enable hostpath-storage multus
+sudo microk8s enable metallb:10.0.0.2-10.0.0.4
+```
+
+Add a Juju model:
+
+```console
+juju add model <model-name>
+```
+
+Initialise the provider:
+
+```console
+terraform init
+```
+
+Customize the configuration inputs under `terraform.tfvars` file according to requirement.
+
+Replace the values in the `terraform.tfvars` file:
```

Comment:
```suggestion
Fill in the mandatory config options in the `terraform.tfvars` file:
```
I feel like having both these lines is a bit redundant. WDYT?

Reply:
I think we can replace.